core/lib/dal/src/blocks_dal.rs

```diff
@@ -402,17 +402,18 @@ impl BlocksDal<'_, '_> {
         number_range: ops::RangeInclusive<L1BatchNumber>,
         eth_tx_id: u32,
         aggregation_type: AggregatedActionType,
-    ) -> DalResult<()> {
+    ) -> anyhow::Result<()> {
```

Comment:
please keep dal error

Reply:
Well... I can't. There is no possibility to customize the DalResult. So it's either anyhow or panic

Reply:
@slowli what is the best approach for it? 
Probably we could add custom errors to dal error?

Reply:
maybe it makes sense panicking here as we've discussed it should panic anyway.

Reply:
Just for me, it's better to panic, than using anyhow... 
But i think we have an ability to customize the errors

Reply:
`Instrumented::constraint_error` fits here

Reply:
Oh, thank you @perekopskiy! That's what I wanted